scripts/config-dark.json

```diff
@@ -136,9 +136,9 @@
         }
       ]
     },
-    "CSSOpacityCustomProp--darkerties": {
```

Comment:
What is this change used for?

Reply:
I made this change as the naming for this group was incorrect and not reflecting the name of the default tokens used for light theming. The name in the default tokens is `CSSOpacityCustomProperties` and this should be `CSSOpacityCustomProperties--dark`.